entries/p/paypal.com.json

```diff
@@ -4,12 +4,13 @@
     "url": "https://www.paypal.com/",
     "tfa": [
       "sms",
-      "totp"
+      "totp",
+      "u2f",
+      "custom-software"
```

Comment:
Since the only custom/proprietary 2FA software is Symantec VIP which, as you said, is deprecated if not entirely removed by now, I don't think `custom-software` should be listed.